src/ripple/app/main/Main.cpp

```diff
@@ -552,9 +549,6 @@ run(int argc, char** argv)
             std::cout << "VACUUM beginning. page_size: " << pageSize
                       << std::endl;
 
-            session << "PRAGMA journal_mode=OFF;";
-            session << "PRAGMA temp_store_directory=\"" << tmpPath.string()
```

Comment:
The purpose of the temp_store_directory is to make sure there's enough space to perform a VACUUM, which essentially requires as much as the entire database being VACUUMed. The temp_store in memory will likely cause a system to run out of RAM. Imagine transaction DB with 2TB+. (https://sqlite.org/tempfiles.html)

Reply:
Good catch! I'll change that.